source/dataman/AMDbUpgrade1Pt6.cpp

```diff
@@ -0,0 +1,141 @@
+#include "AMDbUpgrade1Pt6.h"
+
+#include "util/AMErrorMonitor.h"
+
+#include <QStringBuilder>
+
+AMDbUpgrade1Pt6::AMDbUpgrade1Pt6(const QString &databaseNameToUpgrade, QObject *parent)
+	: AMDbUpgrade(databaseNameToUpgrade, parent)
+{
+
+}
+
+AMDbUpgrade1Pt6::~AMDbUpgrade1Pt6()
+{
+
+}
+
+QStringList AMDbUpgrade1Pt6::upgradeFromTags() const
+{
+	return QStringList();
+}
+
+bool AMDbUpgrade1Pt6::upgradeNecessary() const
+{
+	return databaseToUpgrade_->tableExists("VESPERSXRFScanConfiguration_table")
```

Comment:
Good eye.  There is actually no code to drop tables in any of the database upgrade system.  In this case, the upgrade won't happen every time because the AMDbUpgrades table keeps track of which upgrades have happened by comparing the tags.  So we're safe with a call like this.